engine/runtime/src/test/scala/org/enso/std/test/BooleanTest.scala

```diff
@@ -12,6 +12,7 @@ class BooleanTest extends InterpreterTest {
     "support if_then_else" in {
       val code =
         """from Standard.Builtins import all
+          |from Standard.Base.Data.Boolean import all
```

Comment:
So I'm wondering if this doesn't make these tests many times slower? If it does, we probably should move these to Enso and only leave non-builtins-dependent tests in scala.

Reply:
Haven't measured the effect of my changes on tests running. I will leave it for the end, once everything is moved to get a better idea. Good point, though!

Reply:
So yeah, the tests take longer, I'm afraid. I added timings option to scalatest to visualize the problem.
Some of the unit tests could be moved to enso-defined test-suite but others, like runtime instrumentation ones, not really. 
But I would prefer not to do the former in this PR as it is already big enough.